doc/075_scripting.rst

```diff
@@ -83,13 +83,23 @@ JSON output of most restic commands are documented here.
     list of allowed values is documented may be extended at any time.
 
 
+Exit errors
+-----------
+
+Fatal errors will result in a final JSON message before the process exits.
```

Comment:
It might be interesting to add a remark that JSON output cannot be guaranteed if parsing the CLI arguments failed.

Reply:
I added this comment:

```
.. note::
    Some errors cannot be caught and reported this way,
    such as Go runtime errors or command line parsing errors.
```